### PR TITLE
Add hello world program

### PR DESCRIPTION
**Pull request by JakeRoggenbuck**

### CODE REVIEW

File: examples/hello_world.cz [hunk 1]
```diff
@@ -0,0 +1,5 @@
+with sys
+
+fun hello () {
```
Comment (adamhutchings) [inline examples/hello_world.cz]: This may be a style thing, but maybe `fun hello() {`

File: examples/hello_world.cz [hunk 2]
```diff
@@ -0,0 +1,5 @@
+with sys
```
Comment (adamhutchings) [inline examples/hello_world.cz]: Should we need to import sys?

Comment (JakeRoggenbuck) [reply examples/hello_world.cz]: I thought that was your idea to use stdin and stdout

File: examples/hello_world.cz (hunk 1 above)
Comment (JakeRoggenbuck) [reply examples/hello_world.cz]: fixed it

File: examples/hello_world.cz (hunk 2 above)
Comment (adamhutchings) [reply examples/hello_world.cz]: Yeah, it was, but maybe `sys` could be implicitly imported

Comment (JakeRoggenbuck) [reply examples/hello_world.cz]: sure, sounds good

File: examples/hello_world.cz [hunk 3]
```diff
@@ -0,0 +1,3 @@
+fun hello() {
```
Comment (adamhutchings) [inline examples/hello_world.cz:2]: Sorry to keep requesting changes, but you never call `hello` anywhere so this program doesn't do anything.

Comment (JakeRoggenbuck) [reply examples/hello_world.cz:2]: yup, got it

**Review by JakeRoggenbuck — COMMENTED** [review]
Also added comments

**Pull request merged**